File: server/src/app/api/userRoutes.ts
import express from 'express';
import { createUser } from '@server/app/services/db/userService';

const router = express.Router();

router.post('/createUser', async (req, res) => {
    try {
        const { name } = req.body;
        const user = await createUser(name);
        res.json({ result: { message: `Hello ${user.name}, your account has been created!` } });
    } catch (error) {
        console.error('Error creating user:', error);
        res.status(500).json({ error: 'Failed to create user' });
    }
});

export default router;

File: server-fast/app/api/userRoutes.py
from fastapi import APIRouter, HTTPException
from ..services.db.userService import create_user

router = APIRouter()

@router.post("/createUser")
async def create_user_endpoint(name: str):
    try:
        user = await create_user(name)
        return {"result": {"message": f"Hello {user['name']}, your account has been created!"}}
    except Exception as e:
        raise HTTPException(status_code=500, detail="Failed to create user")
    